rocprofiler-systems/Jacobi/README.md

```diff
@@ -0,0 +1,228 @@
+# Rocprof-sys
+NOTE: extensive documentation on how to use rocprof-sys for the [GhostExchange examples](https://github.com/amd/HPCTrainingExamples/tree/main/MPI-examples/GhostExchange) is also available as `README.md` in this exercises repo. Here, we show how to use rocprof-sys tools considering the example in HPCTrainingExamples/HIP/jacobi.
+
+In this series of examples, we will demonstrate profiling with rocprof-sys on a platform using an AMD Instinct&trade; MI300 GPU, hosted in AAC environment. ROCm releases (6.3.2) include rocprof-sys.
```

Comment:
Let's say MI300A here too.

Reply:
Also, correct "ROCm releases (6.3.2) include rocprof-sys." to "ROCm 6.3.2 release includes the `rocprofiler-systems` package that you can install."